recipes/quill/all/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.13.5)
```

Comment:
The reason for this is that I create the install target in a subdirectory not in the root project directory.
https://stackoverflow.com/questions/34443128/cmake-install-targets-in-subdirectories

Please advice if you think it is not required to enforce the cmake version

Reply:
so please, remove it. Conan CMake helper should be able to solve it.

Reply:
thanks, reverted to previous